.gitignore

```diff
@@ -3,4 +3,14 @@ public/lib
 .sass-cache/
 .DS_store
 c4h_keys.txt
+<<<<<<< HEAD
```

Comment:
You failed to clean this file, make sure all the merge conflict diff are removed